src/session/content/message_row/sticker.rs

```diff
@@ -86,10 +90,16 @@ impl MessageSticker {
                     }),
                 );
 
-                message
-                    .chat()
-                    .session()
-                    .download_file(data.sticker.sticker.id, sender);
+                    let session = self
+                        .ancestor(Content::static_type())
+                        .unwrap()
```

Comment:
I'm getting a crash here for some reason.

Reply:
I think what's happening here is that the `update_widget()` gets called so early that it isn't even in the widget tree. This can happen because `update_widget()` is called on the message notify signal, which is set in `constructed()`, in which the widget isn't still fully created yet.